ui/pages/first-time-flow/create-password/new-account/new-account.component.js

```diff
@@ -0,0 +1,232 @@
+            onClick={(e) => {
+              e.preventDefault();
+              this.props.history.push(INITIALIZE_SELECT_ACTION_ROUTE);
+            }}
+            href="#"
+          >
+            {`< ${t('back')}`}
+          </a>
+        </div>
+        <div className="first-time-flow__header">{t('createPassword')}</div>
+        <form className="first-time-flow__form" onSubmit={this.handleCreate}>
+          <TextField
+            data-testid="create-password"
+            id="create-password"
+            label={t('newPassword')}
+            type="password"
+            className="first-time-flow__input"
+            value={password}
+            onChange={(event) => this.handlePasswordChange(event.target.value)}
+            error={passwordError}
+            autoFocus
+            autoComplete="new-password"
+            margin="normal"
+            fullWidth
+            largeLabel
+          />
+          <TextField
+            data-testid="confirm-password"
+            id="confirm-password"
+            label={t('confirmPassword')}
+            type="password"
+            className="first-time-flow__input"
+            value={confirmPassword}
+            onChange={(event) =>
+              this.handleConfirmPasswordChange(event.target.value)
+            }
+            error={confirmPasswordError}
+            autoComplete="confirm-password"
+            margin="normal"
+            fullWidth
+            largeLabel
+          />
+          <div
+            className="first-time-flow__checkbox-container"
+            onClick={this.toggleTermsCheck}
+          >
+            <div
+              className="first-time-flow__checkbox"
+              tabIndex="0"
+              role="checkbox"
+              onKeyPress={this.onTermsKeyPress}
+              aria-checked={termsChecked}
+              aria-labelledby="ftf-chk1-label"
+            >
+              {termsChecked ? (
+                <Icon name={ICON_NAMES.CHECK} size={SIZES.XL} />
+              ) : null}
+            </div>
+            <span
+              id="ftf-chk1-label"
+              className="first-time-flow__checkbox-label"
+            >
+              {t('acceptTermsOfUse', [
+                <a
+                  onClick={(e) => e.stopPropagation()}
+                  key="first-time-flow__link-text"
+                  href="https://metamask.io/terms.html"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  <span className="first-time-flow__link-text">
+                    {t('terms')}
+                  </span>
+                </a>,
+              ])}
+            </span>
+          </div>
+          <Button
+            type="primary"
+            className="first-time-flow__button"
+            disabled={!this.isValid() || !termsChecked}
+            onClick={this.handleCreate}
+          >
+            {t('create')}
+          </Button>
+        </form>
+      </div>
+    );
+  }
+}
```

Comment:
You shouldn't be adding an entire file here? Maybe a rebase mistake?

Reply:
The file in the develop branch
<img width="705" alt="image" src="https://user-images.githubusercontent.com/53999998/218238510-d9e32bcd-50c0-40db-accc-07d2a9a733e5.png">

The file in my branch
<img width="967" alt="image" src="https://user-images.githubusercontent.com/53999998/218238538-1ea9457f-45b7-47d5-ab4f-5a72d01889bb.png">


Maybe the folder structure is causing the issue. Please suggest what can be done in this case @georgewrmarshall

Reply:
Moreover, I have 2 of the same file in my branch because of the merge.

<img width="923" alt="image" src="https://user-images.githubusercontent.com/53999998/218238631-a52dd278-c810-4172-b6e8-9f2eaa5002d8.png">

Reply:
@georgewrmarshall should I just remove this file and edit the other one?

Reply:
Yes, I believe so